core/ops.rs

```diff
@@ -20,19 +22,84 @@ use std::ops::DerefMut;
 use std::pin::Pin;
 use std::rc::Rc;
 
-pub type OpAsyncFuture = Pin<Box<dyn Future<Output = Box<[u8]>>>>;
-pub type OpFn = dyn Fn(Rc<RefCell<OpState>>, BufVec) -> Op + 'static;
+pub use erased_serde::Serialize as Serializable;
```

Comment:
Is this temporary or will it be required to use `erased_serde` after all the refactors are done?

Reply:
None of the op implementors will need to use `erased_serde`, but we need it to Box serializable stuff so async ops can then serialize it with `serde_v8`.

Right now it's used for sync & async ops, for sync ops I'll introduce a faster approach (saving the `Box` alloc overhead)